UnitTests/GraphViewTests.cs

```diff
@@ -52,12 +52,27 @@ public override void DrawAxisLabel (GraphView graph, int screenPosition, string
 
 	public class GraphViewTests {
 
+		static string LastInitFakeDriver;
 
 		public static FakeDriver InitFakeDriver ()
 		{
 			var driver = new FakeDriver ();
-			Application.Init (driver, new FakeMainLoop (() => FakeConsole.ReadKey (true)));
+			try {
```

Comment:
I dig all this. At some point I'd like to extract these test-helper-code things out of `GraphViewTests` into a test library.